src/generator/operations.ts

```diff
@@ -128,17 +128,23 @@ export async function generateOperations(session: Session<CodeModel>): Promise<O
         clientParams.sort(sortParametersByRequired);
         for (const clientParam of values(clientParams)) {
           methodParams.push(`${clientParam.language.go!.name} ${formatParameterTypeName(clientParam)}`);
+          if (clientParam.language.go!.description) {
+            paramDocs.push(`// ${clientParam.language.go!.name} - ${clientParam.language.go!.description}`);
+          }
         }
       }
     }
 
     const methodParams = new Array<string>();
+    const paramDocs = new Array<string>();
     if (<boolean>session.model.language.go!.azureARM) {
       // AzureARM is the simplest case, no parametertized host etc
       imports.add('github.com/Azure/azure-sdk-for-go/sdk/azcore');
       emitClientParams();
       methodParams.push('credential azcore.TokenCredential');
+      paramDocs.push('// credential - the credential used to authenticate the request.');
```

Comment:
Trying to fit in a getting started hint:
```suggestion
      paramDocs.push('// credential - used to authorize requests. Usually a credential from azidentity.');
```

Reply:
Good idea, fixed